pyrobosim/examples/demo.py

```diff
@@ -97,13 +98,24 @@ def create_world(multirobot=False):
         width_clearance=0.01,
         depth_clearance=0.01,
     )
-    robot = Robot(
-        name="robot",
+
+    robot0 = Robot(
+        name="robot0",
         radius=0.1,
         path_executor=ConstantVelocityExecutor(),
         grasp_generator=GraspGenerator(grasp_props),
     )
-    world.add_robot(robot, loc="kitchen")
+    planner_config_astar = {
+        "grid": occupancy_grid_from_world(
+            world, resolution=0.05, inflation_radius=1.5 * robot0.radius
+        ),
+        "diagonal_motion": True,
+        "heuristic": "euclidean",
+    }
```

Comment:
The planner in this demo is A*, but in the YAML file and in the ROS demo it's RRT.

Maybe switch this to also default to RRT with the same settings. But see my bug with the A* path yaws in the main review!